include/api.php

```diff
@@ -130,6 +130,18 @@ function pll__( $string ) {
 }
 
 /**
+ * Translates a string ( previously registered with pll_register_string ) and escapes it for safe use in HTML output.
+ * 
+ * @since 2.1
+ * 
+ * @param string $string the string to translate
+ * @return string translation in the current language
+ */
+function plll_esc_htlm__ ( $string ) {
```

Comment:
Could you please rename the function to pll_esc_html__ including in the function where it is used below?